projects/angular/src/popover/signpost/signpost-content.ts

```diff
@@ -56,6 +56,8 @@ const POSITIONS: string[] = [
   host: { '[class.signpost-content]': 'true', '[id]': 'signpostContentId' },
 })
 export class ClrSignpostContent extends AbstractPopover implements OnDestroy {
+  @Input('clrSignpostCloseAriaLabel') signpostCloseAriaLabel: string;
```

Comment:
Can we just use it as a default value?
```suggestion
  @Input('clrSignpostCloseAriaLabel') signpostCloseAriaLabel = this.commonStrings.keys.signpostClose;
```

Reply:
See my other comment for why defaulting to the common string in the template is better (and consistent with existing code). https://github.com/vmware-clarity/ng-clarity/pull/1624#discussion_r1868210855